go-controller/pkg/ovn/common.go

```diff
@@ -63,8 +63,10 @@ func (oc *Controller) forEachAddressSetUnhashedName(iteratorFn func(
 func addToAddressSet(hashName string, address string) {
 	klog.V(5).Infof("addToAddressSet for %s with %s", hashName, address)
 
+	// IPv6 addresses need to  be quoted, IPv4 work either way.
+	addressQuotes := strconv.Quote(address)
 	_, stderr, err := util.RunOVNNbctl("add", "address_set",
-		hashName, "addresses", address)
+		hashName, "addresses", addressQuotes)
```

Comment:
can you split the quoting stuff (including the corresponding unit test updates) into its own commit, then do the dual-stack stuff as a second commit?

also, you don't really need the temporary variable... you could just change `address` to `strconv.Quote(address)` in the call. Also, you don't *really* need to use `strconv.Quote`... we know the address doesn't contain any quotes or anything else that needs to be escaped, so you could just do `` `"` + address + `"` ``?

Reply:
Will do. I'll move it to another PR so it is easier to manage. Should be posted within the hour.